src/app/v2/trade/[primary]/[numeraire]/page.tsx

```diff
@@ -0,0 +1,80 @@
+'use client';
+
+import { PairSelector } from '@/components/PairSelector';
+import { Chart } from '@/components/chart';
+import { RouteBook } from '@/components/route-book';
+import { Card } from '@penumbra-zone/ui/Card';
+import { useAssets } from '@/shared/state/assets.ts';
+import { useMemo } from 'react';
+
+interface QueryParams {
+  primary: string;
+  numeraire: string;
+}
```

Comment:
Naming proposal for asset1 and asset2

Reply:
I don't know what's industry standard, I have also seen: baseAsset/quoteAsset. To me denominator also sounds more standard than numeraire.